### PR TITLE
fix(curriculum): rename cert

### PR DESCRIPTION
**Pull request by moT01**
This renames the `Front End Development` cert to `Full Stack Developer` cert, along with all the files and folders that go with that - except the `25-front-end-development` folder. I left that so we don't get conflicts on all the curriculum content PR's.

Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

### CODE REVIEW

**Comment by ilenia-magoni** [comment]
This should maybe wait until November, or all the quizzes PRs will have merge conflicts, and merge conflicts because the files moved are a mess to deal with

**Comment by moT01** [comment]
yea, I left that folder for that reason. I think this should be fine and only create minimal conflicts, if any.

**Review by jdwilkin4 — COMMENTED** [review]
There were multiple versions in chat and in huddles about the name.

One of the discussions was to name this `Full Stack Development Certification`
Another conversation was to name this `Certified Full Stack Developer certification`

So we need to nail down the exact name so we are not in a situation where we have to open another PR to update this again 👍

**Comment by moT01** [comment]
This is what I wrote down that Quincy said to use.

**Comment by jdwilkin4** [comment]
I would still feel more comfortable if we double checked on this just because it has changed so many times.

**Comment by moT01** [comment]
TODO:
Use "Certified Full Stack Developer Curriculum" for the main button
Use "Certified Full Stack Developer" for everything else

**Comment by ilenia-magoni** [comment]
Lint is failing for this PR

**Comment by moT01** [comment]
Well, this is going to cause conflicts and/or make the tests fail on open curriculum PR's to the new cert - but we are just going to have to deal with them. I think this is ready to go.

**Review by huyenltnguyen — APPROVED** [review]
I left some small nits and questions, but I think changes could be made in a follow-up PR.

File: tools/challenge-auditor/index.ts [hunk 1]
```diff
@@ -55,7 +55,7 @@ const superBlockFolderMap = {
   'rosetta-code': '22-rosetta-code',
   'python-for-everybody': '23-python-for-everybody',
   'b1-english-for-developers': '24-b1-english-for-developers',
-  'front-end-development': '25-front-end-development'
+  'full-stack-developer': '25-front-end-development'
```
[the remark this answers is not in the document]
Comment (moT01) [reply tools/challenge-auditor/index.ts:58]: yes

File: shared/config/certification-settings.ts [hunk 2]
```diff
@@ -26,7 +26,7 @@ export enum Certification {
   CollegeAlgebraPy = 'college-algebra-with-python-v8',
   FoundationalCSharp = 'foundational-c-sharp-with-microsoft',
   // Upcoming certifications
-  FrontEndDevelopment = 'front-end-development-v9',
+  FullStackDeveloper = 'full-stack-developer-v9',
```
[the remark this answers is not in the document]
Comment (moT01) [reply shared/config/certification-settings.ts:29]: I thought we should just stick with it. That's the format we've been using for a bit now.

File: client/src/pages/learn/full-stack-developer/workshop-cat-photo-app/index.md [hunk 3]
```diff
@@ -1,9 +1,9 @@
...
 title: Introduction to Build a Cat Photo App
 block: workshop-cat-photo-app
-superBlock: front-end-development
+superBlock: full-stack-developer
 ---
 
 ## Introduction to Build a Cat Photo App
 
-This is the first workshop for new frontend cert.
+This is the first workshop for the new full stack developer.
```
[the remark this answers is not in the document]
Comment (moT01) [reply client/src/pages/learn/full-stack-developer/workshop-cat-photo-app/index.md]: ✅

File: client/src/pages/learn/full-stack-developer/index.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+---
+title: Certified Full Stack Developer
+superBlock: full-stack-developer
+certification: full-stack-developer
+---
+
+## Certified Full Stack Developer
+
+Learn HTML, CSS, JavaScript, Relational Databases, React, and Node.js to become a certified full-stack developer.
```
[the remark this answers is not in the document]
Comment (moT01) [reply client/src/pages/learn/full-stack-developer/index.md]: ✅

**Comment by moT01** [comment]
I added the blocked label here because I want to rebase and fix up any curriculum stuff that was added right before this gets merged - to avoid tests failing on main. It could still use some reviews and approvals.

**Review by majestic-owl448 — APPROVED** [review]

**Pull request merged**